packages/core/src/libraries/hook/index.ts

```diff
@@ -54,6 +36,53 @@ export const createHookLibrary = (queries: Queries) => {
     hooks: { findAllHooks, findHookById },
   } = queries;
 
+  const sendWebhooks = async <T extends HookEventPayloadWithoutHookId>(hooks: Hook[], payload: T) =>
+    Promise.all(
```

Comment:
should have concurrency limit?

Reply:
Use pMap to add concurrency control. Also, make this method a flat map of hooks requests.